### ISSUE TITLE
Orgrimmar Interceptor

### ISSUE THREAD

**Issue by Thekid1994** [WoWManiaUK/Redemption#2631]
I messaged a GM and they referred me here. I was just curious why the interceptor isn't a two person flying mount despite the fact it has 2 seats on it.

**Comment by TenmanWoW** [comment]
Well, here my two cents.


The mount is supposed to seat two people, according to wowhead. However, mounts from later expansions (In this case, Legion, I believe) can be imported to our server. However, the bells and whistles (In your case, the two seat ability) may not carry over. I have Heart of the Aspects which, in retail, changes color and speed with ever increasing riding skill/location used. On WoW-Mania, that function has not been coded in, so it's just a really really REALLY pretty mount.

Tl;Dr  We are able to import mounts from later expansions but not all features of those mounts may be available due to compatability/stability/coding required issues (Im guessing at that last part).

I assume the GM referred you here because this is where our coder's would see such issues. I'm not sure it would be possible for them to make that mount a true two seater; that's something they would have to decide.

**Comment by TheManiax** [comment]
As explained, the imported features are As-is.
We tried to import them in the best possible way.
If any developer can change it to better, feel free to re-open the report.
